front/app/modules/commercial/project_description_builder/hooks/useProjectDescriptionBuilderLayout.ts

```diff
@@ -1,24 +1,33 @@
 import { useState, useEffect } from 'react';
+import useProjectById from 'api/projects/useProjectById';
 import {
   projectDescriptionBuilderLayoutStream,
   IProjectDescriptionBuilderLayout,
 } from '../services/projectDescriptionBuilder';
 
 const useProjectDescriptionBuilderLayout = (projectId: string) => {
+  const { data: project } = useProjectById(projectId);
   const [projectDescriptionBuilderLayout, setProjectDescriptionBuilderLayout] =
```

Comment:
@luucvanderzee Was thinking this extra hook call would make the request slower, but I imagine this data will probably be cached somewhere already right?

Reply:
Yeah exactly, in practice this data will already be in the cache